### PR TITLE
Sort files for local development

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,14 +1,35 @@
 version: '3'
 services:
```
Comment (andrewrlee) [inline docker-compose.yml:2]: don't need to worry about commenting this out if you scale the container down 
`docker-compose up --scale help-with-prison-visits-internal=0`

**Review by andrewrlee — APPROVED** [review]
LGTM

File: docker-compose.yml [hunk 2]
```diff
@@ -1,14 +1,35 @@
...
-  help-with-prison-visits-internal:
-    build:
-      context: .
-    volumes:
-      - "./gulpfile.js:/usr/src/app/gulpfile.js"
-      - "./nodemon.json:/usr/src/app/nodemon.json"
-      - "./app:/usr/src/app/app"
-      - "./logs:/usr/src/app/logs"
-      - "./uploads:/usr/src/app/uploads"
+  # help-with-prison-visits-internal:
```
Comment (psoleckimoj) [inline docker-compose.yml:3]: Unnecessary right now, will sort out at a later date

File: helm_deploy/values-prod.yaml [hunk 3]
```diff
@@ -5,7 +5,7 @@ generic-service:
   ingress:
     host: "manage.hwpv.hmpps.service.justice.gov.uk"
   env:
-    APVS_MOJ_SSO_REDIRECT_URI: 'https://manage.hwpv.hmpps.service.justice.gov.uk/auth/hmppsauth/callback'
+    APVS_MOJ_SSO_REDIRECT_URI: 'https://manage.hwpv.hmpps.service.justice.gov.uk/login/callback'
```
Comment (mattops) [inline helm_deploy/values-prod.yaml:8]: is this env var used?

**Pull request merged**